platform/pulp/platform/models/consumer.py

```diff
@@ -0,0 +1,67 @@
+"""
+Django models related to content consumers.
+"""
+
+from django.contrib.contenttypes.fields import GenericRelation
+from django.db.models import TextField, ForeignKey, ManyToManyField, CASCADE
```

Comment:
I'm favor using a "from django.db import models" syntax as well, but not because it's what django does.

The reasons I see for this are:

- 
less maintenance of the Import statement if we need to use another item from "django.db.models"
- 
we maintain context about what we are using.  In many places it can be frustrating to identify where something came from in code without having to scroll back to the import statements.  Having the name-spaced context in front of what is used can help. (simple example: ceil() vs math.ceil() )
 
While this introduce some more up-front cost and can be considered annoying, pointless and tedious, I find it to be of benefit in the future and worth the investment if someone not as familiar with this code can have an easier time understanding it and working with it.